src/applications/personalization/dashboard/utils/appeals-helpers.jsx

```diff
@@ -0,0 +1,248 @@
+import * as Sentry from '@sentry/browser';
+import { format } from 'date-fns';
+
+// This literally determines how many rows are displayed per page on the index page
+export const DECISION_REVIEW_URL = '/decision-reviews';
+
+export const APPEAL_TYPES = {
+  legacy: 'legacyAppeal',
+  supplementalClaim: 'supplementalClaim',
+  higherLevelReview: 'higherLevelReview',
+  appeal: 'appeal',
+};
+
+export const appealTypes = Object.values(APPEAL_TYPES);
+
+export const programAreaMap = {
+  compensation: 'disability compensation',
+  pension: 'pension',
+  insurance: 'insurance',
+  loan_guaranty: 'loan guaranty', // eslint-disable-line camelcase
+  education: 'education',
+  vre: 'vocational rehabilitation and employment',
+  medical: 'health care',
+  burial: 'burial benefits',
+  fiduciary: 'fiduciary',
+};
+
+/**
+ * Returns a string with the formatted name of the type of appeal.
+ * @param {Object} appeal
+ * @returns {string}
+ */
+export function getTypeName(appeal) {
+  switch (appeal.type) {
+    case APPEAL_TYPES.supplementalClaim:
+      return 'supplemental claim';
+    case APPEAL_TYPES.higherLevelReview:
+      return 'higher-level review';
+    case APPEAL_TYPES.legacy:
+    case APPEAL_TYPES.appeal:
+      return 'appeal';
+    default:
+      Sentry.withScope(scope => {
+        scope.setExtra('type', appeal.type);
+        Sentry.captureMessage('appeals-unknown-type');
+      });
+      return null;
+  }
+}
+
+export const DOCKET_TYPES = {
+  directReview: 'directReview',
+  evidenceSubmission: 'evidenceSubmission',
+  hearingRequest: 'hearingRequest',
+};
+
+/**
+ * Returns a string with the formatted name of the AMA docket.
+ * @param {string} docket
+ * @returns {string}
+ */
+export function getDocketName(docket) {
+  switch (docket) {
+    case DOCKET_TYPES.directReview:
+      return 'Direct Review';
+    case DOCKET_TYPES.evidenceSubmission:
+      return 'Evidence Submission';
+    case DOCKET_TYPES.hearingRequest:
+      return 'Hearing Request';
+    default:
+      return docket;
+  }
+}
+
+export const AOJS = {
+  vba: 'vba',
+  vha: 'vha',
+  nca: 'nca',
+  other: 'other',
+};
+
+export function getAojDescription(aoj) {
+  switch (aoj) {
+    case AOJS.vba:
+      return 'Veterans Benefits Administration';
+    case AOJS.vha:
+      return 'Veterans Health Administration';
+    case AOJS.nca:
+      return 'National Cemetery Administration';
+    default:
+      return 'Agency of Original Jurisdiction';
+  }
+}
```

Comment:
A simple comment here about what AOJS is might be useful. Since I have no context, I was confused at first as to what the acronym meant.

Reply:
good call! i had a similar problem and figured it out while going through the `getStatusContents` function, since this was copied over from the claims application.